db/migrate/20170403023324_add_cascade_delete_to_images.rb

```diff
@@ -0,0 +1,7 @@
+class AddCascadeDeleteToImages < ActiveRecord::Migration[5.0]
+  def change
+    remove_foreign_key :images, :listings
+    add_foreign_key :images, :listings, on_delete: :cascade
```

Comment:
@meyerhoferc ya had to do some digging, for whatever reason 'dependent :destroy' wasn't doing the trick